src/client.spec.ts

```diff
@@ -1,1627 +1,1627 @@
+
+  beforeEach(() => {
+    mockOptimizelyUserContext = {
+      decide: jest.fn(),
+      decideAll: jest.fn(),
+      decideForKeys: jest.fn(),
+      fetchQualifiedSegments: jest.fn(),
+      setForcedDecision: jest.fn(),
+      removeForcedDecision: jest.fn(),
+      removeAllForcedDecisions: jest.fn(),
+    } as any;
+
+    mockInnerClient = {
+      createUserContext: jest.fn(() => mockOptimizelyUserContext),
+      activate: jest.fn(() => null),
+      track: jest.fn(),
+      isFeatureEnabled: jest.fn(() => false),
+      getEnabledFeatures: jest.fn(() => []),
+      getVariation: jest.fn(() => null),
+      setForcedVariation: jest.fn(() => false),
+      getForcedVariation: jest.fn(() => null),
+      getFeatureVariableBoolean: jest.fn(() => null),
+      getFeatureVariableDouble: jest.fn(() => null),
+      getFeatureVariableJSON: jest.fn(() => null),
+      getAllFeatureVariables: jest.fn(() => {
+        return {};
+      }),
+      getFeatureVariable: jest.fn(() => null),
+      getFeatureVariableInteger: jest.fn(() => null),
+      getFeatureVariableString: jest.fn(() => null),
+      getOptimizelyConfig: jest.fn(() => null),
+      onReady: jest.fn(() => Promise.resolve({ success: false })),
+      close: jest.fn(),
+      getVuid: jest.fn(),
+      sendOdpEvent: jest.fn(),
+      notificationCenter: {
+        addNotificationListener: jest.fn(() => 0),
+        removeNotificationListener: jest.fn(() => false),
+        clearNotificationListeners: jest.fn(),
+        clearAllNotificationListeners: jest.fn(),
+      },
+    };
+
+    const anyOptly = optimizely as any;
+    anyOptly.createInstance.mockReturnValue(mockInnerClient);
+    createInstanceSpy = optimizely.createInstance as jest.Mock<optimizely.Client, [optimizely.Config]>;
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('provides the initial config object via the initialConfig property', () => {
+    const instance = createInstance(config);
+    expect((instance as MockedReactSDKClient).initialConfig).toEqual(config);
+  });
+
+  it('provides a default user object', () => {
+    const instance = createInstance(config);
+    expect(instance.user).toEqual({
+      id: null,
+      attributes: {},
+    });
+  });
+
+  it('provides access to the underlying client', () => {
+    const instance = createInstance(config);
+    expect(createInstanceSpy).toBeCalledTimes(1);
+    expect(createInstanceSpy.mock.results[0].type).toBe('return');
+    expect(createInstanceSpy.mock.results[0].value).toBe((instance as MockedReactSDKClient).client);
+  });
+
+  it('adds react-sdk clientEngine and clientVersion to the config, and passed the config to createInstance', () => {
+    createInstance(config);
+    expect(createInstanceSpy).toBeCalledTimes(1);
+    expect(createInstanceSpy).toBeCalledWith({
+      ...config,
+      clientEngine: 'react-sdk',
+      clientVersion: '3.0.0-beta',

```

Comment:
This was the only expected change.  I'm not sure why GitHub is showing a lot of change